src/components/Main/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import './Main.css';
+import Modal from './Modal/Modal';
+
+const Main = (props) => {
+  const [modal, setModal] = useState(false);
+
+  return (
+    <div>
+      <h1>Hello, {props.currentUser.name}!</h1>
+
+      <button onClick={() => setModal(true)} className="ui green button">
+        New Task
+      </button>
+
+      {modal ? <Modal setModal={setModal}/> : null}
+    </div>
```

Comment:
Avoid leaving empty lines.

```suggestion
    <div>
      <h1>Hello, {props.currentUser.name}!</h1>
      <button onClick={() => setModal(true)} className="ui green button">
        New Task
      </button>
      {modal ? <Modal setModal={setModal}/> : null}
    </div>
```